lib/compat/wordpress-6.8/functions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Adds the x-wav mime type to the list of mime types.
+ * This is necessary for Firefox as it uses the identifier for uploaded .wav files.
+ *
+ * @since 6.8.0
+ *
+ * @param string[] $mime_types Mime types.
+ * @return string[] Mime types keyed by the file extension regex corresponding to those types.
+*/
+function gutenberg_get_mime_types_6_8( $mime_types ) {
+	if ( isset( $mime_types['wav'] ) ) {
+		$mime_types['wav|x-wav'] = 'audio/wav';
+		unset( $mime_types['wav'] );
+	}
```

Comment:
This works, but maybe the check should be for `x-wav` in case plugins/themes have filtered this in the meantime.

```php
	/*
	 * Only add the mime type if it doesn't already exist
	 * to avoid overriding any customizations.
	 */
	if ( ! isset( $mime_types['x-wav'] ) && ! isset( $mime_types['wav|x-wav'] ) ) {
		$mime_types['x-wav'] = 'audio/wav';
	}
```